frontend/components/Blocks/SectionBlock/SectionBlock.tsx

```diff
@@ -109,17 +135,10 @@ export default function Section({ data }) {
           }
         })}
       </div>
-      <div className="flex flex-col lg:w-1/2">
-        {/* TODO: Set the imagesize dynamically */}
+      <div className={`flex flex-col ${gridValue.right}`}>
         <div className="lg:sticky py-12 px-10 lg:px-16 lg:pt-24 top:0">
           {Object.keys(media).length > 0 && (
-            /* eslint-disable @next/next/no-img-element */
-            <img
-              src={process.env.NEXT_PUBLIC_BASE_URL + media.img?.src}
-              alt={media.img?.alt}
-              width={1600}
-              height={900}
-            />
+            <Image src={media.img?.src} alt={media.img?.alt} width="1600" height="900" />
```

Comment:
Ï see that in the case of several static images uploaded in Wagtail it only renders the last image. Maybe we could limit the option in Wagtail to only upload one static image (as this seems the most likely scenario within a section).